Editor/Credentials/Strings.cs

```diff
@@ -130,5 +130,11 @@ public static class Strings
         public const string SettingsUIDeployNextStepLabel = "SettingsUIDeployNextStepLabel";
         public const string LabelOpenSdkIntegrationDoc = "LabelOpenSdkIntegrationDoc";
         public const string LabelOpenSdkApiDoc = "LabelOpenSdkApiDoc";
+        
+        public const string TabLanding = "TabLanding";
+        public const string TabCredentials = "TabCredentials";
+        public const string TabAnywhere = "TabAnywhere";
+        public const string TabManagedEC2 = "TabManagedEC2";
+        public const string TabHelp = "TabHelp";
```

Comment:
Should this file be moved somewhere else perhaps? This is under `/Editor/Credentials/` but I see a lot of these strings (and previous strings) don't have anything to do with credentials?

Reply:
Moved file up a dir